### PR TITLE
Adjust risk cards story

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
If there was only one exposure (high or low risk), the text "Zuletzt am date" is replaced with "Am date "
The green risk card with low risk, shows the date of the last exposure similar to the red risk card
Refactoring risk data model
Adding a new launch argument for number of days with risk exposure
Adding tests and screenshots

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4838
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4839
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4876
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4895

## Screenshots
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 08 53
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 10 45
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 14 30
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 13 22
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 36 28
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 13 35 47

Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 16 50 28
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 16 51 19
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 16 51 48
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 16 52 24
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-01-29 at 16 52 49

### CODE REVIEW

**Review by nickguendling — DISMISSED** [review]
Looking good! I think the changes for the detail screen are missing (ExposureDetectionViewController/Model)

**Comment by naveeddotio** [comment]
> Looking good! I think the changes for the detail screen are missing (ExposureDetectionViewController/Model)

@nickguendling added the changes. Thanks!

**Review by SabineLoss — APPROVED** [review]
UA reviewed

**Comment by naveeddotio** [comment]
> UA reviewed

Thanks

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift [hunk 1]
```diff
@@ -656,10 +656,18 @@ class ExposureDetectionViewModelTests: XCTestCase {
 	) {
 		// Risk data section
 		var section = dynamicTableViewModel.section(0)
-		XCTAssertEqual(section.cells.count, 3)
-		XCTAssertEqual(section.cells[0].cellReuseIdentifier.rawValue, "riskCell")
-		XCTAssertEqual(section.cells[1].cellReuseIdentifier.rawValue, "riskCell")
-		XCTAssertEqual(section.cells[2].cellReuseIdentifier.rawValue, "riskCell")
+		if hasAtLeastOneDayWithLowRiskLevel == false {
```
[the remark this answers is not in the document]
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift]: @marcussc thanks for looking, I was going with the wind :D

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift [hunk 2]
```diff
@@ -346,16 +346,28 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
...
 		return DynamicTableViewModel([
-			riskDataSection(
-				footer: .riskTint(height: 16),
-				cells: [
-					.riskContacts(text: AppStrings.Home.riskCardLowNumberContactsItemTitle, image: UIImage(named: "Icons_KeineRisikoBegegnung")),
-					.riskStored(activeTracing: activeTracing, imageName: "Icons_TracingCircle-Dark_Step %u"),
-					.riskRefreshed(text: AppStrings.ExposureDetection.refreshed, image: UIImage(named: "Icons_Aktualisiert"))
-				]
-			),
+			numberOfExposures > 0 ? riskDataSectionWithExposure : riskDataSectionWithOutExposure,
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift:370]:
```suggestion
			numberOfExposures > 0 ? riskDataSectionWithExposure : riskDataSectionWithoutExposure,
```

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Risk/HomeRiskCellModel.swift [hunk 3]
```diff
@@ -265,6 +265,38 @@ class HomeRiskCellModel: CountdownTimerDelegate {
 
 		let activeTracing = risk.details.activeTracing
 
+		let mostRecentDateWithLowRisk = risk.details.mostRecentDateWithRiskLevel
+		var formattedMostRecentDateWithLowRisk = ""
+		if let mostRecentDateWithLowRisk = mostRecentDateWithLowRisk {
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Risk/HomeRiskCellModel.swift:270]:
```suggestion
		if let mostRecentDateWithLowRisk = risk.details.mostRecentDateWithRiskLevel {
```

**Review by ArturFriesen — APPROVED** [review]

**Review by tcfos — APPROVED** [review]

**Pull request merged**